src/client/Microsoft.Identity.Client/ApiConfig/AbstractPublicClientAcquireTokenParameterBuilder.cs

```diff
@@ -21,6 +23,23 @@ internal AbstractPublicClientAcquireTokenParameterBuilder(IPublicClientApplicati
             PublicClientApplicationExecutor = publicClientApplicationExecutor;
         }
 
+        public AbstractPublicClientAcquireTokenParameterBuilder<T> WithPoPAuthenticationScheme(Uri uri) // TODO: naming and comments
+        {
+            CommonParameters.AddApiTelemetryFeature(ApiTelemetryFeature.WithScheme);
```

Comment:
do we want to know which scheme in the telemetry?  Perhaps "WithPoPScheme"?  (and we can look to make WithBearerScheme the default)